src/index.html

```diff
@@ -6,9 +6,54 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Handjet:wght@200;400;700&family=Roboto:wght@500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
+    <header class="header">
+      <a href="index.html" class="logo">
+        <img
+        src="images/logo.png"
+        alt="logo"
+        class="logo__img"
+      >
+      </a>
+      <nav class="nav">
+        <ul class="nav__links">
+          <li>
+            <a class="nav__link is-active" href="#">APPLE</a>
+          </li>
+          <li>
+            <a class="nav__link" href="#">SAMSUNG</a>
+          </li>
+          <li>
+            <a class="nav__link" href="#">SMARTPHONES</a>
+          </li>
+          <li>
+            <a
+            class="nav__link"
+            href="#"
+            id="pc"
+            data-qa="hover">
+              LAPTOPS & COMPUTERS
+            </a>
```

Comment:
The same about formatting.
Also, why did you add `id`, and what does it mean "pc"?